README.md

```diff
@@ -0,0 +1,40 @@
+# Portfolio
+
```

Comment:
- [ ] **[OPTIONAL]** You have done a job making the header logo into a link.

![alej_m](https://user-images.githubusercontent.com/36542986/133743385-9b02c43a-a596-4e1a-bada-1caed9886985.jpg)

However, for conventional design reasons, it would be nice to remove the underline property of the logo link, by giving the link itself a class or id and styling it like so:
```
text-decoration: none;
```

Reply:
Thanks for your suggestion!!
I really appreciate it! 👨‍💻